docs/source/gbd2017_models/causes/covid_19/index.rst

```diff
@@ -11,35 +11,285 @@ Covid-19 Custom SEIR Vivarium Cause Model
+   * - :math:`N`
+     - Number of individuals in all model states
+   * - :math:`\beta(t)`
+     - Degree of transmission (fit to/driven by model covariates). Approximately average number of contacts per person per time multiplied by the disease transmission probability of a contact between an infectious and susecptible individual
+   * - :math:`\alpha`
+     - Parameter to account for cluster nature of population transmission network. Ex: if alpha < 1, when the infectious population is large  the force of infection is attenuated
+   * - :math:`\sigma`
+     - Rate at which an exposed person becomes infectious (enters the :math:`I_1` state), i.e. :math:`1/\text{duration}_{E}`
+   * - :math:`\gamma_1`
+     - Rate at which an individual transitions from :math:`I_1` to :math:`I_2`, i.e. :math:`1/\text{duration}_{I_{1}}`
+   * - :math:`\gamma_2`
+     - Rate at which an infectious individual (:math:`I_2`) recovers, i.e. :math:`1/\text{duration}_{I_{2}}`
+
+.. todo::
+
+  Obtain precise definition of :math:`I_1` versus :math:`I_2`
+
+.. note::
+
+	In standard SEIR models, the parameter :math:`\beta(t)` defined here is often a *constant* parameter that represents the average number of contacts per person per time multiplied by the transmission probability of a contact between an infectious and susceptible individual (equal to :math:`R_0` when unit time is equal to the duration of the infectious period).
+
+	However, as described above, the :math:`\beta(t)` parameter is a function of time the considers the time-varying covariates used in the IHME forecasting model, which allows for the consideration of changes in relative mobility (which could be considered a proxy variable for the number of social contacts per person per time).
+
 Vivarium Modeling Strategy
 --------------------------
 
+Modeling covid-19 in Vivarium will require a different strategy than our standard Vivarium cause models. The standard methodology for Vivarium cause models assumes event independence in that one simulant's disease incidence does not affect another simulant's chance of disease incidence. However, given the infectious nature of covid-19, this cause model document will outline a strategy to model the dependent infectious nature of the disease transmission.
+
 Scope
 +++++
 
-Assumptions and Limitations
-+++++++++++++++++++++++++++
+The immediate intention of this cause model will be to aid in modeling various scenarios relating to varied population density, mobility, and other potential covid-19 control measures at the University of Washington and immediately surrounding areas/populations. This covid-19 Vivarium cause model is intended to make use of the data and framework of the existing IHME covid-19 forecast model SEIR component (briefly described above). The scope of this cause model document is intended to develop a basic strategy for an infectious SEIR cause model and will start with minimal complexity and can be expanded upon if it becomes necessary.
```

Comment:
I know you currently don't know, and I'm sure you were already planning on it, but as you develop a clearer idea, it would be good to list out the scenarios of interest here.

Even right now, it might be good to list categories of what it would be _possible_ for us to model. Specifically, would we consider doing sub-populations which do not mix homogeneously with one another?